os-updateSecret.sh

```diff
@@ -4,9 +4,9 @@
 
 #set -x
 
-# if not arguments then print help
-if [ $# -eq 0 ]; then
-    echo "$0: <SECRET_NAME> <DIRECTORY> "
+# Must have both arguments.
```

Comment:
Is this script likely to be used in other projects?  If so, it shouldn't be part of this project.  

It belongs in [tl-its-umich-edu/DockerUtility](/tl-its-umich-edu/DockerUtility) instead.

Reply:
If the group agrees on a specific repo to use and, more importantly, to look at, I'll happily put it there.  I've put out a number of scripts already that are public and ignored.

Reply:
That repo seems to be the right place.  Between the two of us, if you say that's the right place for our Docker tools, that's good enough for me.

I'm not sure who's been ignoring it.  I've done very little at all with Docker.  I've only recently started using it for LC and John Pariseau of C/E has supplied all the necessary configuration files.  I think we should continue putting useful Docker tools there and referring the rest of group to it.  That's the way I started the caliper-event-analysis repo, too.